django/contrib/auth/middleware.py

```diff
@@ -100,12 +98,14 @@ def _remove_invalid_user(self, request):
         but only if the user is authenticated via the RemoteUserBackend.
         """
         try:
-            stored_backend = load_backend(request.session.get(auth.BACKEND_SESSION_KEY, ''))
+            stored_backend = auth.load_backend(
+                request.session.get(auth.BACKEND_SESSION_KEY, '')
```

Comment:
This one is fine as `auth.load_backend()` is used in `clean_username()` above.

```suggestion
                request.session.get(auth.BACKEND_SESSION_KEY, ''),
```